sds_data_manager/constructs/ialirt_ingest_lambda_construct.py

```diff
@@ -62,20 +62,35 @@ def create_dynamodb_table(self) -> aws_dynamodb.Table:
             # Restore data to any point in time within the last 35 days.
             # TODO: change to True in production.
             point_in_time_recovery=False,
-            # Partition key (PK) = Mission Elapsed Time (MET).
+            # Partition key (PK) = ingest year (YYYY).
```

Comment:
Why are you making this change? Just want some backstory

Reply:
Apologies. I added more info in the Overview above.